### DISCUSSION TITLE
Not working

### DISCUSSION THREAD

**Discussion by verjammt** [nxvvvv/safe-exam-browser-bypass#6] [General]
i did what it said in the README but when starting safe exambrowser starts normally and not with my desktop, so idk what to do

**Comment by nxvvvv** [comment]
Could you explain a bit more? I'm not getting you.

**Comment by verjammt** [comment]
> Could you explain a bit more? I'm not getting you.

do i have to have this VMmare player downloaded?

**Comment by nxvvvv** [comment]
This patch fundamentally alters Safe Exam Browser's perception, preventing it from recognizing when it operates within a VMware or virtual machine environment. Without applying this patch, the software fails to load within a virtual machine. To successfully run SEB in a virtual environment, you need to follow these steps:

1. Install VMware Player.
2. Create a [Windows](https://go.microsoft.com/fwlink/p/?linkid=2195587&clcid=0x409&culture=en-us&country=us) virtual machine within VMware Player.
3. Install Safe Exam Browser within the Windows virtual machine.
4. Apply this patch to ensure proper functionality.

By following this process, you can seamlessly use Safe Exam Browser within a virtual machine environment.

**Comment by verjammt** [comment]
> This patch fundamentally alters Safe Exam Browser's perception, preventing it from recognizing when it operates within a VMware or virtual machine environment. Without applying this patch, the software fails to load within a virtual machine. To successfully run SEB in a virtual environment, you need to follow these steps:
> 
> 1. Install VMware Player.
> 2. Create a [Windows](https://go.microsoft.com/fwlink/p/?linkid=2195587&clcid=0x409&culture=en-us&country=us) virtual machine within VMware Player.
> 3. Install Safe Exam Browser within the Windows virtual machine.
> 4. Apply this patch to ensure proper functionality.
> 
> By following this process, you can seamlessly use Safe Exam Browser within a virtual machine environment.

how do i create this windows virtual machine

**Comment by nxvvvv** [comment]
Wait a few moments, I'll guide you .

**Comment by verjammt** [comment]
i need a operating system

**Comment by nxvvvv** [comment]
### Step 1: Download and Install VMware Player

1. Visit the official VMware website (https://www.vmware.com/products/workstation-player.html) to download VMware Player.
2. Follow the on-screen instructions to install VMware Player on your computer.

### Step 2: Download Windows 10 ISO

Download the Windows 10 Enterprise LTSC 2021 ISO from the provided link: [Windows 10 Enterprise LTSC 2021 ISO](https://drive.massgrave.dev/en-us_windows_10_enterprise_ltsc_2021_x64_dvd_d289cf96.iso)

### Step 3: Create a New Virtual Machine

1. Open VMware Player after installation.
2. Click on "Create a New Virtual Machine."

### Step 4: Virtual Machine Wizard

1. Choose "Installer disc image file (iso)" and click "Browse" to select the Windows 10 ISO file you downloaded.
2. Click "Next."

### Step 5: Guest Operating System

1. Select "Microsoft Windows" as the guest operating system.
2. Choose the version as "Windows 10 x64."
3. Click "Next."

### Step 6: Name and Location

1. Enter a name for your virtual machine (e.g., "Windows 10 VM").
2. Choose a location to save the VM files.
3. Click "Next."

### Step 7: Disk Capacity

1. Specify the disk capacity for the virtual machine (e.g., 40GB recommended).
2. Select "Store virtual disk as a single file."
3. Click "Next."

### Step 8: Ready to Create Virtual Machine

1. Review your settings and click "Finish."

### Step 9: Customize VM Settings

1. Before starting the VM, click "Edit virtual machine settings."
2. In the Hardware tab, you can adjust settings such as memory, processors, and add/remove hardware if needed.

### Step 10: Start the Virtual Machine

1. Click "Play virtual machine" to start the Windows 10 installation process.

### Step 11: Install Windows 10

1. Follow the on-screen instructions to install Windows 10.
2. When prompted, enter your product key and configure other settings.

### Step 12: Complete Installation

1. Complete the Windows setup process.
2. Install VMware Tools for better integration and performance (click "Player" > "Manage" > "Install VMware Tools").

Congratulations! You've successfully installed Windows 10 on a virtual machine using VMware Player. 

### Step 13: Install and patch SEB

1. Download and install SEB from [here](https://sourceforge.net/projects/seb/files/seb/SEB_3.5.0/SEB_3.5.0.544_SetupBundle.exe/download)
2. Download the patch files from [here](https://github.com/nxvvvv/safe-exam-browser-bypass/releases/tag/v3.5.0.544)
3. Replace the three files : [SafeExamBrowser.SystemComponents.dll](https://github.com/nxvvvv/safe-exam-browser-bypass/releases/download/v3.5.0.544/SafeExamBrowser.SystemComponents.dll),[SafeExamBrowser.Monitoring.dll](https://github.com/nxvvvv/safe-exam-browser-bypass/releases/download/v3.5.0.544/SafeExamBrowser.Monitoring.dll) and [SafeExamBrowser.Client.exe](https://github.com/nxvvvv/safe-exam-browser-bypass/releases/download/v3.5.0.544/SafeExamBrowser.Client.exe) in C:\Program Files\SafeExamBrowser\Application
4. Done.

Now you can write your exam inside the VMware and search for the materials outside VMware, basically, on your normal browser.

Try this and ask if anything arrives

**Comment by verjammt** [comment]
can i also use win 11?

**Comment by verjammt** [comment]
and if possible ciould you help with the recommended part i didnt foudn that eithe rit wasnt in the document part of my pc

**Comment by nxvvvv** [comment]
> can i also use win 11?

Yes you can use Windows 11. I just provided Windows 10 LTSC since it's very lightweight.

> and if possible ciould you help with the recommended part i didnt foudn that eithe rit wasnt in the document part of my pc

Could you explain a bit more? If you meant this: C:\Program Files\SafeExamBrowser\Application, you'll see that only after installing Safe Exam Browser initially

**Comment by verjammt** [comment]
in you README document theres a part called RECOMMENDED when i searched for tha ti ddint find where to edit this code

**Comment by nxvvvv** [comment]
Ah, I understand now. Essentially, you need to complete the full OS installation. After that, navigate to the 'Documents' folder and then proceed to 'Virtual Machines.' You should find the newly installed Windows there. If the folder is named 'LTSC,' simply open that folder, and the VMX file you need to check will be named 'LTSC.vmx.'

**Comment by verjammt** [comment]
oh so i do that after i did everything you said before

**Comment by nxvvvv** [comment]
![image](https://github.com/nxvvvv/safe-exam-browser-bypass/assets/34748927/ac3d40f8-6256-4d12-a60d-637bb3872c84)

**Comment by nxvvvv** [comment]
check that part for the location. go there and edit it

**Comment by nxvvvv** [comment]
Is everything working? May I close the discussion? @verjammt
